src/main/scala/viper/gobra/frontend/info/implementation/resolution/NameResolution.scala

```diff
@@ -304,6 +304,31 @@ trait NameResolution { this: TypeInfoImpl =>
       case n => symbTableLookup(n)
     }
 
+  private def getAllEmbedded(scope: PUnorderedScope, int: PInterfaceType): Vector[PInterfaceType] = {
+    val allEmbedded = int.embedded.map[Option[Vector[PInterfaceType]]](emb => {
+      val embt = tryLookup(sequentialDefenv(scope), emb.typ.id.name)
```

Comment:
I did not do anything about imported types because when trying to add a member to an interface belonging to a different package I get the ANTL error:

```
14:30:46.399 [main] ERROR viper.gobra.GobraRunner$ - An unknown Exception was thrown.
14:30:46.400 [main] ERROR viper.gobra.GobraRunner$ - ANTLR threw unexpected errorsTranslation of interfaceType interface{test.bar
f()
} failed: Imported types are not yet supported as embedded fields. viper.gobra.frontend.ParseTreeTranslator.$anonfun$visitInterfaceType$1(ParseTreeTranslator.scala:414)
java.lang.Exception: ANTLR threw unexpected errorsTranslation of interfaceType interface{test.bar
f()
} failed: Imported types are not yet supported as embedded fields. viper.gobra.frontend.ParseTreeTranslator.$anonfun$visitInterfaceType$1(ParseTreeTranslator.scala:414)
	at viper.gobra.frontend.Parser$.parseSource$1(Parser.scala:84)
	at viper.gobra.frontend.Parser$.$anonfun$parseSources$9(Parser.scala:150)
	at scala.collection.immutable.Vector1.map(Vector.scala:1886)
	at scala.collection.immutable.Vector1.map(Vector.scala:375)
	at viper.gobra.frontend.Parser$.parseSources(Parser.scala:151)
	at viper.gobra.frontend.Parser$.parse(Parser.scala:48)
	at viper.gobra.Gobra.performParsing(Gobra.scala:239)
	at viper.gobra.Gobra.$anonfun$verify$3(Gobra.scala:152)
	at scala.util.Either.flatMap(Either.scala:352)
	at viper.gobra.Gobra.$anonfun$verify$1(Gobra.scala:150)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:672)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:431)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

So I thought that for now we wouldn't do something about that since it is more general. Should I try to include this feature?